### DISCUSSION TITLE
reaload waybar?

### DISCUSSION THREAD

**Discussion by se7en-x230** [prasanthrangan/hyprdots#1088] [Q&A] [answered]
As I have only a few changes I decided to script my setup update with a git pull ... ./install.sh -r
and then basically copy things back the way I like it.
All works fine but I have trouble with the waybar reloading

![image](https://github.com/prasanthrangan/hyprdots/assets/69777713/dc1974de-9819-437a-865b-506bdaa0c60f)

I thought the killall waybar || waybar would read the config.ctl new an set my waybar back the way I like it. 
I does BUT I have to use Super+Alt+arrows to "reload it. 
Any idea why this step is necessary and how I can avoid it?

**Comment by kRHYME7** [comment]
There is another script that writes the config.json and style.css on the file. so killing then rerunning waybar is useless. I'ts either change that ``` killall waybar || waybar ``` part with ``` ~/.config/hyprdots/scripts/wbarconfgen.sh ```

**Reply by se7en-x230** [reply, marked as answer]
your the best thanks
